strabo/static/js/map.js

```diff
@@ -37,7 +37,24 @@ $(document).ready(function(){
     //set_map_click(map);
 
     set_geolocation(map);
+
+});
+
+$(document).keyup(function(e) {
+    if (e.keyCode==27) { //if ESC key is hit
```

Comment:
What I was taught in my systems class about this is that if there is some information in a comment that can be in the executed code itself, it should be--not because it is more clear, or easier to read, or less complicated, but simply because many people don't pay any attention to the comments, and so they go out of date and become useless (or worse) really fast.

Not really an issue here, I don't think. I think that problem is mainly caused by experienced programmers navigating through the code too fast. Just thought I would bring that point up.